extensions/cornerstone/src/init.tsx

```diff
@@ -39,11 +40,27 @@ export default async function init({
   configuration,
   appConfig,
 }: Types.Extensions.ExtensionParams): Promise<void> {
-  await cs3DInit();
+  await cs3DInit({
```

Comment:
can you pull new changes? I added some modification to cs3DInit recently, and it makes sense to merge those changes here as well